packages/insomnia/src/ui/components/dropdowns/preview-mode-dropdown.tsx

```diff
@@ -41,6 +43,64 @@ export const PreviewModeDropdown: FC<Props> = ({
     copyToClipboard();
   }, [copyToClipboard]);
 
+  const exportAsHAR = async () => {
+    if (!response || !request || !isRequest(request)) {
+      console.warn('Nothing to download');
```

Comment:
question: you removed a comment:

```suggestion
      // Should never happen
      console.warn('Nothing to download');
```

I'm curious why?  An FC refactor like this isn't entirely the place to get into it, but I find the comment more helpful than the console warn because it's more precise about what went wrong.

Reply:
I find this style of superfluous commentary distracting because it indicates an expectation that it could go wrong but doesn't tell me why.